libs/crypto/ed25519/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+add_library(ed25519 SHARED
+    add_scalar.c
+    fe.c
+    ge.c
+    key_exchange.c
+    keypair.c
+    sc.c
+    seed.c
+    sha3.c
+    sign.c
+    verify.c
+    )
+target_include_directories(ed25519 INTERFACE .)
```

Comment:
So that current directory (libs/crypto/ed25519) will be appended to include directories of target, which will be linked to `ed25519`. This allows to include without full path, like `#include <ed25519.h>`, which is now used in other files